deployments/common/crds/k8s.nginx.org_virtualserverroutes.yaml

```diff
@@ -608,9 +612,11 @@ spec:
                 externalEndpoints:
                   type: array
                   items:
-                    description: ExternalEndpoint defines the IP and ports used to connect to this resource.
+                    description: ExternalEndpoint defines the IP/ Hostname and ports used to connect to this resource.
```

Comment:
Minor detail here. We can remove the space between the `/` and `Hostname` in this description.

Reply:
There is likely only going to be either IP or Hostname reported in the status (there may be cases where both are reported for a LoadBalancer service but I think it would be rare?) which is why I included the slash.